src/search-modal/FilterByBlockType.jsx

```diff
@@ -27,6 +27,39 @@ const FilterByBlockType = () => {
   } = useSearchContext();
   // TODO: sort blockTypes first by count, then by name
```

Comment:
@yusuf-musleh Could you remove this. I think is no longer necesary

Reply:
Done, removed.